cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/repo.go

```diff
@@ -436,6 +439,14 @@ func (s *Server) validateUserManagedRepoSecret(
 					if secret.Data["keyFile"] == nil || secret.Data["certFile"] == nil {
 						return nil, status.Errorf(codes.Internal, "Specified secret [%s] missing fields 'keyFile' and/or 'certFile'", secretRef)
 					}
+				case corev1.PackageRepositoryAuth_PACKAGE_REPOSITORY_AUTH_TYPE_DOCKER_CONFIG_JSON:
```

Comment:
Great, I'm updating my PR (#4954) to also support it. Thanks!

Reply:
Thank you for review